content/en/security_platform/cloud_workload_security/workload_security_rules.md

```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
+
+    {{< img src="security_platform/cws/workload_security_rules/define_runtime_expression.png" alt="Adding a rule to the expression field" >}}
+
+5. Define the logic for when this rule triggers a security signal. For example, `a>0` means a security signal triggers as long as the rule condition set in step 4 is met at least once in the sliding time window. Select a severity to associate the rule with and select all relevant parties you to notify.
+
+    {{< img src="security_platform/cws/workload_security_rules/rule_cases.png" alt="Setting a rule trigger, severity, and notification" >}}
+
+6. Name the rule and add the actual notification message in markdown format. It is recommended to include a remediation runbook in the body. As noted in the template, use substitution variables to dynamically generate contextualized content at runtime. After the message, add multiple tags to give more context to the signals generated by your custom rule.
+
+7. Save the rule. This automatically navigates you back to the main Rules page. Click the **Download Workload Security Policy** button in the top right corner to download a `default.policy` file to your local machine.
+
+Complete the remaining steps based on your environment:
+
+{{< tabs >}}
+{{% tab "Host" %}}
+
+8. Copy the `default.policy` file over to the target host, in the `{$DD_AGENT}/runtime-security.d` folder. Ensure the file minimally has `read` and `write` access for the `dd-agent` user on the host.
+
+    **Note:** This may require use of a utility such as SCP or FTP.
+
+9. Restart the [Datadog Agent][1].
+
+
+[1]: /agent/guide/agent-commands/?tab=agentv6v7#restart-the-agent
+{{% /tab %}}
+
+{{% tab "Helm" %}}
+
+8. Create a configmap containing `default.policy`. For example, `kubectl create configmap jdefaultpol --from-file=default.policy`.
+
+9. Add the configMap (`jdefaultpol`) into values.yaml, `datadog.securityAgent.runtime.policies.configMap`:
+
+    ```yaml
+    securityAgent:
+      compliance:
+        # [...]
+      runtime:
+        # datadog.securityAgent.runtime.enabled
+        # Set to true to enable Security Runtime Module
+        enabled: true
+        policies:
+          # datadog.securityAgent.runtime.policies.configMap
+          # Place custom policies here
+    configMap: *jdefaultpol*
+      syscallMonitor:
+        # datadog.securityAgent.runtime.syscallMonitor.enabled
+        # Set to true to enable Syscall monitoring.
+        enabled: false
+    ```
+
+10. Upgrade the helm chart with the following command: `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`
+
+    **Note:** If you need to make further changes to `default.policy` you can either use `kubectl edit cm jdefaultpol` or replace the configMap with  `kubectl create configmap jdefaultpol --from-file default.policy -o yaml --dry-run=client | kubectl replace -f -`
+
+{{% /tab %}}
+{{< /tabs >}}
+
```

Comment:
@sarina-dd do you intend on adding a Further Reading section to this page?

Reply:
Good catch! Let's leave this as-is for now, and once we roll out the new auto-generated CWS rules page, I'll add a further reading section with a link to that page.